tests/blackbox/stratis_cli_cert.py

```diff
@@ -156,23 +158,29 @@ def test_pool_create_same_name(self):
         Test creating a pool that already exists.
         """
         self.unittest_command(
-            [STRATIS_CLI, "pool", "create", make_test_pool(), DISKS[0]], 1, False, True
+            [STRATIS_CLI, "pool", "create", make_test_pool(DISKS[0:1]), DISKS[1]],
+            1,
+            False,
+            True,
         )
 
     def test_pool_add_cache(self):
         """
         Test adding cache to a pool.
         """
         self.unittest_command(
-            [STRATIS_CLI, "pool", "add-cache", make_test_pool(), DISKS[1]], 0, 1, True
+            [STRATIS_CLI, "pool", "add-cache", make_test_pool(DISKS[0:1]), DISKS[1]],
```

Comment:
I'ld like you to make the number of disk that you create the pool with larger than the number of disks that you add to the pool for a cache. So, something like this:

```
[STRATIS_CLI, "pool", "add-cache", make_test_pool(DISKS[0:2]), DISKS[2]]
```